aksel.nav.no/website/pages/_app.tsx

```diff
@@ -15,6 +15,10 @@ function App({ Component, pageProps, router }: AppProps) {
     window.location.host === "design.nav.no" &&
       window.location.replace(`http://aksel.nav.no`);
 
+    !window.location.pathname.startsWith("/templates/") &&
```

Comment:
For å unngå at resten av siden påvirkes kan denne kanskje flippes 🤔 
`if templates or examples body.classlist.remove("globalstyles")`
`globalstyles` må da legges til som default på body også da evt

Reply:
Det gir mening